arduino-ide-extension/src/browser/dialogs/version-welcome-dialog.tsx

```diff
@@ -0,0 +1,93 @@
+import React from '@theia/core/shared/react';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { Message } from '@theia/core/shared/@phosphor/messaging';
+import { ReactDialog } from '../theia/dialogs/dialogs';
+import { nls } from '@theia/core';
+import { DialogProps } from '@theia/core/lib/browser';
+import { WindowService } from '@theia/core/lib/browser/window/window-service';
+import { AppService } from '../app-service';
+
+@injectable()
+export class VersionWelcomeDialogProps extends DialogProps {}
+
+@injectable()
+export class VersionWelcomeDialog extends ReactDialog<void> {
+  @inject(AppService)
+  private readonly appService: AppService;
+
+  @inject(WindowService)
+  private readonly windowService: WindowService;
+
+  constructor(
+    @inject(VersionWelcomeDialogProps)
+    protected override readonly props: VersionWelcomeDialogProps
+  ) {
+    super({ title: '' });
+
+    this.node.id = 'version-welcome-dialog-container';
+    this.contentNode.classList.add('version-welcome-dialog');
+  }
+
+  protected render(): React.ReactNode {
+    return (
+      <div>
+        <p>
+          {nls.localize(
+            'arduino/versionWelcome/donateMessage',
+            'Arduino is committed to keeping software free and open-source for everyone. Your donation helps us develop new features, improve libraries, and support millions of users worldwide.'
+          )}
+        </p>
+        <p className="bold">
+          {nls.localize(
+            'arduino/versionWelcome/donateMessage2',
+            'Please consider supporting our efforts to keep Arduino IDE free.'
```

Comment:
```suggestion
            'Please consider supporting our work on the free open source Arduino IDE.'
```

The previous message implies there is a plan to make the project non-free. That is not an appropriate tone to take in our communications to the users.